### ISSUE TITLE
Feature: record avatar animations

### ISSUE THREAD

**Issue by sirkitree** [MetaGenAI/MetaGenNeos#16]
# Observation
Animations on the avatar are not currently recorded.

# Request
Record animations that are happening on the avatar during recording for playback.
- this could be limited to just the avatar lighting up when speaking, such as with the default Neos headset avatar and possibly expand to encompass more later

**Comment by xela-zone** [comment]
the most robust way to do this would likely be to record the audio stream and controller inputs for playback.

**Comment by guillefix** [comment]
Yeah I'm already recording and "reinstalling" the audio stream in the "Debug play" option of the system. I do plan to record the controller inputs but I haven't finished looking at how to do that.

But what you are asking could be done by just looking for all the Fields inside the avatar. However, before doing that, we would need to update the recording system to a system that only records changes in values, otherwise the animation files would be way too big.
